chromium/rules.js

```diff
@@ -101,7 +101,7 @@ RuleSet.prototype = {
   apply: function(urispec) {
     var returl = null;
     // If we're covered by an exclusion, go home
-    if (this.exclusions !== null) {
+    if (this.exclusions) {
```

Comment:
Why was this changed?

Reply:
Well, is that a bug? What exactly do we want to catch? `null`, `undefined`, anything else?

Reply:
Any falsey or non-array value I guess.